.github/workflows/validate.yml

```diff
@@ -118,14 +118,14 @@ jobs:
         uses: ./.github/actions/setup
 
       - name: Run common tests
-        run: ./gradlew copyBrandingToCommonResources :composeApp:iosSimulatorArm64Test
+        run: ./gradlew copyBrandingToCommonResources :composeApp:desktopTest
```

Comment:
We would have to expand the scpope of these workflows to run all platform tests individually.

Reply:
I see, that makes the CI run faster